weaviate/util.py

```diff
@@ -79,6 +79,44 @@ def image_encoder_b64(image_or_image_path: Union[str, BufferedReader]) -> str:
     return base64.b64encode(content).decode("utf-8")
 
 
+def file_encoder_b64(file_or_file_path: Union[str, BufferedReader]) -> str:
```

Comment:
do we have any test using this?

Reply:
There are two for `image_encode_b64` and `image_decoder_b64`. I can duplicate them for the `file_` functions that will be tested on `integration/weaviate-logo.png`. We would have to source some suitable test files for other media types to test them all out too.